iodata/formats/gamess.py

```diff
@@ -0,0 +1,144 @@
+# IODATA is an input and output module for quantum chemistry.
+# Copyright (C) 2011-2019 The IODATA Development Team
+#
+# This file is part of IODATA.
+#
+# IODATA is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# IODATA is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>
+# --
+"""GAMESS punch file format."""
+
+
+import numpy as np
+
+from ..docstrings import document_load_one
+from ..utils import angstrom, LineIterator
+
+
+__all__ = []
+
+
+PATTERNS = ['*.gamessout']
+
+
+def _read_data(gms) -> tuple:
+    """Extract ``title``, ``symmetry`` and ``symbols`` from the punch file."""
+    title = next(gms).strip()
+    symmetry = next(gms).split()[0]
+    if symmetry != "C1":
+        raise NotImplementedError("Only C1 symmetry is supported.")
+    symbols = []
+    line = True
+    while line != " $END      \n":
+        line = next(gms)
+        if line[0] != " ":
+            symbols.append(line.split()[0])
+    return title, symmetry, symbols
+
+
+def _read_coordinates(gms, res) -> tuple:
+    """Extract ``numbers`` and ``coordinates`` from the punch file."""
+    for i in range(2):
+        next(gms)
+    N = len(res["symbols"])
+    # if the data are already read before, just overwrite them
+    numbers = res.get("atnums")
+    if numbers is None:
+        numbers = np.zeros(N, int)
+        res["atnums"] = numbers
+
+    coordinates = res.get("atcoords")
+    if coordinates is None:
+        coordinates = np.zeros((N, 3), float)
+    for i in range(N):
+        words = next(gms).split()
+        numbers[i] = int(float(words[1]))
+        coordinates[i, 0] = float(words[2]) * angstrom
+        coordinates[i, 1] = float(words[3]) * angstrom
+        coordinates[i, 2] = float(words[4]) * angstrom
```

Comment:
The same feature @tovrstra mentioned below would be applicable here as well.
```suggestion
        coordinates = words[2:5] * angstrom
```